app/_how-tos/allow-multiple-authentication.md

```diff
@@ -45,8 +73,139 @@ cleanup:
       icon_url: /assets/icons/gateway.svg
 ---
 
-@todo
+## 1. Create an ID
+
+Create a UUID:
+
+```
+uuidgen
+```
+
+Export the ID to an environment variable:
+```sh
+export DECK_ANONYMOUS_CONSUMER=434772ef-af0c-4227-a33b-76e33b9fd7df
+```
+
+## 2. Create Consumers
+
+Create three Consumers, including an `anonymous` Consumer.
+The `anonymous` Consumer doesn't correspond to any real user, and will only serve as a fallback:
+
+{% entity_examples %}
+entities:
+  consumers:
+    - username: anonymous
+      id: ${anonymous_consumer}
+    - username: Dana
+    - username: Mahan
+variables:
+  anonymous_consumer:
+    value: $ANONYMOUS_CONSUMER
+{% endentity_examples %}
+
+We're going to assign a different authentication type to each Consumer later.
+
+## 3. Set up authentication
+
+Add the Key Auth and Basic Auth plugins to the `example-service` Gateway Service, and set the `anonymous` fallback to the Consumer we created earlier:
+
+{% entity_examples %}
+entities:
+  plugins:
+    - name: key-auth
+      service: example-service
+      config:
+        hide_credentials: true
+        anonymous: ${anonymous_consumer}
+    - name: basic-auth
+      service: example-service
+      config:
+        hide_credentials: true
+        anonymous: ${anonymous_consumer}
+variables:
+  anonymous_consumer:
+    value: $ANONYMOUS_CONSUMER
+{% endentity_examples %}
+
+## 4. Test with anonymous Consumer
+
+You now have authentication enabled on the Gateway Service, but the `anonymous` Consumer also allows requests from unauthenticated clients.
+
+Check without credentials:
```

Comment:
```suggestion
The following validation works because the anonymous Consumer currently allows this unauthenticated request:
```

Reply:
I wanted a bit more info here about how the anonymous Consumer was allowing this to function.